README.md

```diff
@@ -94,6 +94,31 @@ POST http://my-api.com/endpoint
 
 Type `jsonr --help` for more details on usage once you have a tool installed.
 
+### JavaScript Object Bodies (--js flag)
```

Comment:
Let's remove this from readme. It's not critical feature worth mentioning in readme file